auth/src/test/java/org/aerogear/mobile/auth/authenticator/OIDCAuthenticatorImplTest.java

```diff
@@ -205,4 +234,20 @@ public void onError(Throwable error) {
         authenticator.handleAuthResult(intent);
     }
 
+    @Test
+    public void testLogout() {
+        authenticator.logout(userPrincipalImpl, new Callback<UserPrincipal>() {
```

Comment:
This is an asynchronous method so the callback isn't being called.  Therefor this test isn't testing anything.  You can verify by adding a fails in the success block.